frontend/syntaxmeets/src/components/SyntaxEditor/SyntaxEditor.js

```diff
@@ -85,20 +85,18 @@ const SyntaxEditor = (props) => {
   const [codeError, setCodeError] = useState("");
   // This will resend a message to update the code of the newly joined user
   useEffect(() => {
-
     if (props.previousUser.id === props.id) {
-
       props.socket.emit("message", value);
     }
-  }, [props.previousUser]);
+  }, []);
 
```

Comment:
There is a small mistake, you removed props.previousUser, due to which code is not updated for a new user when he joins, Please rectify it 🙏

Reply:
Oh yes, thanks for letting me know. Fixed it. 🔨